src/msw/window.cpp

```diff
@@ -2577,6 +2579,38 @@ wxMenu* wxWindowMSW::MSWFindMenuFromHMENU(WXHMENU hMenu)
     return nullptr;
 }
 
+wxMenuItem* wxWindowMSW::MSWFindMenuItemFromHMENU(WXHMENU hMenu, int item)
+{
+    MENUITEMINFO mii;
+    mii.cbSize = sizeof(MENUITEMINFO);
```

Comment:
Minor, but this is a bit shorter and more consistent with the rest of our code:
```suggestion
    WinStruct<MENUITEMINFO> mii;
```